src/Command/FrontendCommands.php

```diff
@@ -167,6 +175,48 @@ public function frontendInstall()
             return $result;
         }
         $this->io()->success('Front-end dependencies installed.');
+        $result = $this->taskExec("drush en -y dkan_js_frontend")->dir("docroot")->run();
+        if ($result->getExitCode() != 0) {
+            $this->io()->error('Could not install front-end node module');
+            return $result;
+        }
+        
+        $this->installTheme($opts['theme']);
+        
+        $this->taskExec("drush config-set system.site page.front \"/home\" -y")->run();
+        $this->io()->success('Set front page.');
+    }
+
+    private function installTheme($theme)
```

Comment:
```suggestion
    /**
     * Install the given Drupal theme.
     *
     * @param string $theme
     *   Name of Drupal theme to install.
     */
    private function installTheme(string $theme)
```
Just adding a docblock here.

Reply:
See latest @clayliddell